src/components/ProductCard/ProductCard.tsx

```diff
@@ -27,9 +27,6 @@ const ProductCard = ({ product }: Props) => {
           <span className="font-bold tracking-wide uppercase block mb-0.5">
             {product.brand_name}
           </span>
-          <p>
```

Comment:
Why delete this code?

Reply:
I use ProductCard when I map the products on the page. I didn't want the same category name and id mentioned multiple times on the page so I only mention it once, as a title ( in categories/[categoryId]/page)

Reply:
@damdiana don't forget about this question

Reply:
ok